server/container_remove.go

```diff
@@ -30,7 +31,7 @@ func (s *Server) RemoveContainer(ctx context.Context, req *types.RemoveContainer
 		// The RemoveContainer RPC is idempotent, and must not return an error
 		// if the container has already been removed. Ref:
 		// https://github.com/kubernetes/cri-api/blob/c20fa40/pkg/apis/runtime/v1/api.proto#L74-L75
-		if errors.Is(err, truncindex.ErrNotExist) {
```

Comment:
Have you looked at why checking for a wrapped error here does not work?

The call stack for this error is rather straightforward.

Reply:
I agree the wrapping should technically work :thinking: